lib/Command/FixEncryptedVersion.php

```diff
@@ -57,36 +57,60 @@ protected function configure() {
 				'user',
 				InputArgument::REQUIRED,
 				'The id of the user whose files need fixing'
+			)->addOption(
+				'path',
+				'p',
+				InputArgument::OPTIONAL,
+				'Limit files to fix with path, e.g., --path="/Music". If path indicates a directory, all the files inside directory will be fixed.'
```

Comment:
```suggestion
				'Limit files to fix with path, e.g., --path="Music/Artist". If path indicates a directory and/or mountpoint/directory, all the files inside directory will be fixed.'
```
This is quite nice, because you also can restrict to a mountpoint too which behaves as a directory. We have the same for the files:scan command...
I have also removed the leading "/" as this a bit confusing as mandatory element (even it gets removed later in the code). I also will drop a note regarding docs in a separate comment

Reply:
I'd prefer to keep the leading "/", such as "/Music/Artist":
* It serves as a remainder that you have to deal with it.
* It makes clearer that we're asking for a full path.

As for the mountpoint, I think it's implicit since there is no reason not to allow going inside the mountpoint. I'd expect a warning otherwise.
It's completely fine to include the mention in the docs, as well as provide additional examples, because you can be more explicit there, but for the command help I'd prefer to condense as much information as possible in the least space.

Anyway, that's a personal opinion. I'm Ok with any choice.